js/app/modules/selection/xapian.js

```diff
@@ -56,6 +56,14 @@ const Xapian = new Module.Class({
         throw new Error('You should be implementing construct_query_object in your subclass');
     },
 
+    _modify_query_object(query) {
```

Comment:
Are you running the system 1.47.x version of GJS? This syntax is allowed in 1.48 which is what we shipped with our runtime. We should be using it for new code.